x/slashing/internal/keeper/infractions.go

```diff
@@ -31,20 +30,29 @@ func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr crypto.Address, p
 	index := signInfo.IndexOffset % k.SignedBlocksWindow(ctx)
```

Comment:
it would be even better to load SignedBlockWindow and MinSignedBlocksWindow outside of the loop, and reuse for all validators. I had that change somewhere in older branch, and wanted to submit once this is merged